.cicd/Jenkinsfile

```
@@ -148,6 +148,14 @@ pipeline {
                         }
                     }
 
+                    // Try a few Workflow Task scripts to make sure E2E tests can be launched in a follow-on 'Test' stage
+                    stage('Functional WorkflowTaskTests') {
+                        steps {
+                            echo "Running simple workflow script task tests on ${env.SRW_PLATFORM} (using ${env.WORKSPACE})"
+                            sh 'bash --login "EXPTDIR=${WORKSPACE}/expt_dirs/test_community ${WORKSPACE}/.cicd/scripts/srw_ftest.sh"'
```

The Jenkins tests are failing on this line.  Within the `srw_ftest.sh` script, the `EXPTDIR` is set to `${workspace}/expt_dirs/test_community`.  I think that the EXPTDIR entry will need to be removed.

```suggestion
                            sh 'bash --login ${WORKSPACE}/.cicd/scripts/srw_ftest.sh"'
```

---

sure. update committed.

---

Thanks, @BruceKropp-Raytheon!  I have resubmitted the Jenkins tests.  Resolving this conversation now.